config/staging.yaml

```
@@ -165,4 +165,7 @@ federationRedirect:
     ovh:
       weight: 0
     turing:
-      weight: 0
+      url: https://binder-staging.mybinder.turing.ac.uk
+      weight: 4
+      health: https://binder-staging.mybinder.turing.ac.uk
+      versions: https://binder-staging.mybinder.turing.ac.uk
```

```suggestion
      health: https://binder-staging.mybinder.turing.ac.uk/health
      versions: https://binder-staging.mybinder.turing.ac.uk/versions
```

Correcting the URLs for the health and versions endpoints